### ISSUE TITLE
Clarification: which JS implementation to use?

### ISSUE THREAD

**Issue by dbalatero** [jetify-com/typeid-js#11]
I saw there's two implementations of TypeID linked:

- [official version](https://github.com/jetify-com/typeid-js)
- [community version](https://github.com/ongteckwu/typeid-ts)

Is there any guidance on which one to use, and why you might choose one over the other?

**Comment by LucilleH** [comment]
Hi @dbalatero, at a glance both implementations use Typescript, but the method signatures are different. The official version is maintained by the Jetify Team, as we also maintain the overall [typeid](https://github.com/jetify-com/typeid) spec. 

There's a v3 spec coming out addressing https://github.com/jetify-com/typeid/issues/7, and we will follow and update the library accordingly in the upcoming releases. I'm not super aware of the community version, but the official version will stay up-to-date with the overall spec.

**Comment by dbalatero** [comment]
Makes sense! It's probably worth updating the README to clarify. Is this something you'd want a PR for, or would you prefer to have more control over the style/formatting of the README and do it yourself?

**Comment by elieven** [comment]
Bump?

**Comment by LucilleH** [comment]
Hi all, the official implementation supports v3 spec. The community one is still on v2. If you'd like to use the newest spec, use the official one.
